### PR TITLE
Extend example_dag for Dataproc Batch operators

### PR DESCRIPTION
**Pull request by MaksYermak**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by lwyszomi** [comment]
looks good, please send PR to the community

File: airflow/providers/google/cloud/example_dags/example_dataproc.py [hunk 1]
```diff
@@ -178,13 +179,26 @@
...
 }
+BATCH_CONFIG_WITH_PHS = {
+    "spark_batch": {
+        "jar_file_uris": ["file:///usr/lib/spark/examples/jars/spark-examples.jar"],
+        "main_class": "org.apache.spark.examples.SparkPi",
+    },
+    "environment_config": {
+        "peripherals_config": {
+            "spark_history_server_config": {
+                "dataproc_cluster": f"projects/{PROJECT_ID}/regions/{REGION}/clusters/phs-cluster-name"
```
Comment (lwyszomi) [inline airflow/providers/google/cloud/example_dags/example_dataproc.py:197]: Do we need to create cluster manually? What if I will take this example DAG and I will execute it? Operator which use this config will fail or pass?

Comment (MaksYermak) [reply airflow/providers/google/cloud/example_dags/example_dataproc.py:197]: If you run this config, operator will be fail. I will try to create cluster automatically and then I will change this PR

Comment (lwyszomi) [reply airflow/providers/google/cloud/example_dags/example_dataproc.py:197]: 👍

**Pull request closed without merging**